api/src/test/java/org/openmrs/api/db/hibernate/HibernateConditionDAOTest.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.api.db.hibernate;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Date;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.openmrs.CodedOrFreeText;
+import org.openmrs.Concept;
+import org.openmrs.ConceptName;
+import org.openmrs.Condition;
+import org.openmrs.api.db.ConditionDAO;
+import org.openmrs.ConditionClinicalStatus;
+import org.openmrs.ConditionVerificationStatus;
+import org.openmrs.Patient;
+import org.openmrs.test.BaseContextSensitiveTest;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class HibernateConditionDAOTest extends BaseContextSensitiveTest {
```

Comment:
Others might suggest otherwise but by convention we have the tests for the service layer and not the DAO, the only time we have them for the DAO is for methods that are never reflected in the service layer to be tested. In any case this would be a ConditionDAOTest instead of HibernateConditionDAOTest